UefiTestingPkg/AuditTests/PagingAudit/UEFI/DxePagingAuditTestApp.inf

```diff
@@ -21,15 +21,18 @@
   Dxe/App/DxePagingAuditTestApp.h
   PagingAuditCommon.c
   PagingAuditCommon.h
+  Dxe/App/ProjectMuFunctionalityNull.c

```

Comment:
Normally, a common API is defined by a library class.

The instance with no functionality is the "null" instance and the other instances contain functionality. Can that be used here?